superbench/benchmarks/micro_benchmarks/tensorrt_inference_performance.py

```diff
@@ -88,31 +92,37 @@ def _preprocess(self):
 
         self.__bin_path = str(Path(self._args.bin_dir) / self._bin_name)
 
+        exporter = torch2onnxExporter()
         for model in self._args.pytorch_models:
-            if hasattr(torchvision.models, model):
-                torch.onnx.export(
-                    getattr(torchvision.models, model)(pretrained=True).cuda(),
-                    torch.randn(self._args.batch_size, 3, 224, 224, device='cuda'),
-                    f'{self.__model_cache_path / (model + ".onnx")}',
-                )
-                self._commands.append(
-                    ' '.join(
-                        filter(
-                            None, [
-                                self.__bin_path,
-                                None if self._args.precision == 'fp32' else f'--{self._args.precision}',
-                                f'--batch={self._args.batch_size}',
-                                f'--iterations={self._args.iterations}',
-                                '--workspace=1024',
-                                '--percentile=99',
-                                f'--onnx={self.__model_cache_path / (model + ".onnx")}',
-                            ]
-                        )
-                    )
-                )
-            else:
+            if not (exporter.check_torchvision_model(model) or exporter.check_benchmark_model(model)):
```

Comment:
If one of the models is missing, the whole benchmark will return false?

Reply:
yes, if the config is incorrect I think we should fail it directly?